pkg/istio-agent/sds-agent.go

```diff
@@ -149,17 +150,13 @@ var (
 	gatewaySecretChan       chan struct{}
 )
 
-// SDSAgent contains the configuration of the agent, based on the injected
+// Agent contains the configuration of the agent, based on the injected
 // environment:
 // - SDS hostPath if node-agent was used
 // - /etc/certs/key if Citadel or other mounted Secrets are used
 // - root cert to use for connecting to XDS server
 // - CA address, with proper defaults and detection
-type SDSAgent struct {
-	// Location of JWTPath to connect to CA. If empty, SDS is not possible.
-	// If set SDS will be used - either local or via hostPath.
-	JWTPath string
-
+type Agent struct {
```

Comment:
SDSAgent is more specific

Reply:
This is part of the larger change - where this will include more than SDS. 

The package is 'istio-agent'.